test/harry/main/org/apache/cassandra/harry/SchemaSpec.java

```diff
@@ -182,7 +182,7 @@ public String compile()
             appendWith.run();
             if (shouldAppendAnd)
                 sb.append(" AND");
-            sb.append(" transactional_mode = '").append(options.transactionalMode()).append('\'');
+            sb.append(" transactional_mode = '").append(options.transactionalMode().toString()).append('\'');
```

Comment:
feel free not to use, but `org.apache.cassandra.service.consensus.TransactionalMode#asCqlParam` exists that does `transactional_mode = '<name>'`